multi-agent/tests/integration/endpoints/test_builtins_endpoints.py

```
@@ -0,0 +1,387 @@
+configured.
+"""
+from unittest.mock import Mock
+
+from mas.resources.errors import (
+    BuiltInWriteProtectedError,
+    BuiltinConfigUnavailableError,
+    BuiltinDependentsPublicError,
+)
+
+
+def _fake_resource_dump(**overrides):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_fake_resource_dump`

(ANN202)